boards/arm/b_u585i_iot02a/b_u585i_iot02a-common.dtsi

```diff
@@ -155,13 +155,6 @@
 		data-rate = <OSPI_DTR_TRANSFER>;
 		four-byte-opcodes;
 		status = "okay";
-		sfdp-bfp = [
```

Comment:
Can you also fix `size` whichapparently should be size = <DT_SIZE_M(64)*8> ?

Reply:
converting to Bits